image/templates/helm/stackrox-secured-cluster/templates/openshift-monitoring.yaml

```diff
@@ -1,6 +1,6 @@
 {{- include "srox.init" . -}}
 
-{{- if ._rox.enableOpenShiftMonitoring -}}
+{{- if ._rox.monitoring.openshift.enabled -}}
```

Comment:
Can we move these resources to a shared location, since they are identical for Central / secured cluster?

Reply:
I think we should do refactor once everything else will be working